bundles/automation/org.eclipse.smarthome.automation.module.timer/build.properties

```diff
@@ -2,5 +2,6 @@ output.. = target/classes/
 bin.includes = META-INF/,\
                .,\
                ESH-INF/,\
-               NOTICE
+               NOTICE,\
+               OSGI-INF/org.eclipse.smarthome.automation.module.timer.factory.TimerModuleHandlerFactory.xml
```

Comment:
Shouldn't we use `OSGI-INF/` here? We usually added the whole directory regardless how much files will be part of.

Reply:
Yes.